### DISCUSSION TITLE
Packaged Release?

### DISCUSSION THREAD

**Discussion by CraigRegester** [intersystems/git-source-control#117] [Q&A] [answered]
Hey folks - been a couple months since a packaged release. 

Wondered if there was a plan to get on a similar pattern of releases like the VSC objectscript extension? Tertiary Dot releases like 1.2.1 or similar for bug fixes and minor improvements. 

Alternatively, as I'm not too familiar with zpm, can I pull down the latest commits to main branch using zpm yet without a packaged release?

**Comment by CraigRegester** [comment]
Answered part of this myself experimenting with zpm tonight - the primary question in the second paragraph still stands for @isc-tleavitt and @isc-svelury but for others interested in getting the latest and greatest with zpm (who may not be as versed with zpm like me):

1. Clone main branch using git or download the zip of the main branch from GitHub
2. If using the zip, you may need to unzip (zpm can apparently load an archive like tgz but didn't like the .zip GitHub gave me when I tried to reference it on AIX.)
3. Launch zpm in the desired namespace and execute

`load /path-containing-extracted-dir/git-source-control-main`

To verify, personally, I just found one of the most recent commits that wasn't in 1.2.0 and verified the latest commit was in the file/class I was reviewing (it was.)

**Comment by isc-tleavitt** [comment, marked as answer]
@CraigRegester the short answer is that our automated testing isn't where we want it yet, and because of that I want to spend some dedicated time testing prior to another official release. @isc-svelury has been working on a ton of new features and improvements and we definitely want to get them into your hands once we've done that final round of testing.

**Reply by CraigRegester** [reply]
Absolutely fair and figured that was generally the case given the desire to back this as the official solution from InterSystems w/ support and such. I'm happy with my [approach ](https://github.com/intersystems/git-source-control/discussions/117#discussioncomment-2118763)to pull the latest and greatest I see @isc-svelury putting in and happy to provide feedback on it! Appreciate you both.

**Reply by isc-tleavitt** [reply]
re: versioning, we're using semver but probably won't do patches for older major/minor versions (unless there's significant demand, which I find unlikely).

**Reply by CraigRegester** [reply]
Agree, with this sort of thing it makes sense to roll forward only imo. It's foundational stuff - like a utility addon - and needs to remain agile to changes dictated by it's dependencies (e.g., Git, potentially GitHub/GitLab/others, OpenSSL, SSH, and so on.)
